### PR TITLE
Update README.md for gradle builds

### PR DESCRIPTION
**Pull request by rosen-vladimirov**
Update README in order to describe new sys requirements.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -126,50 +134,68 @@ If you have installed Chocolatey, you can complete these steps to set up JDK, Ap
...
 
 	For example: PATH=...;...;C:\Users\MyUser\AppData\Local\Android\android-sdk\tools;C:\Users\MyUser\AppData\Local\Android\android-sdk\platform-tools
-1. To update the Android SDK to 19 or later, run the following command.
+1. To update the Android SDK to 21 or later, run the following command.
 
 	```Shell
 	android update sdk
 	```
-1. Select all packages for the Android 19 SDK and any other SDKs that you want to install, click **Install** and wait for the installation to complete.
+1. Select all packages for the Android 21 SDK and any other SDKs that you want to install, click **Install** and wait for the installation to complete.
```
Comment (teobugslayer) [inline README.md]: Shameless plug for my super-awesome script:

```
echo yes | android update sdk --filter tools,platform-tools,android-22,android-17,build-tools-22.0.1,sys-img-x86-android-22,extra-android-m2repository,extra-google-m2repository,extra-android-support --all --no-ui
```

File: README.md [hunk 2]
```diff
@@ -69,24 +69,26 @@ You can install and run the NativeScript CLI on Windows or OS X.
 > On Windows systems, you can develop, build, and deploy NativeScript projects that target Android.
 
 * Windows Vista or later
-* [Node.js 0.10.26][Node.js 0.10.26] or a later stable official release
+* [Node.js 0.10.35][Node.js 0.10.35] or a later stable official release
```
Comment (teobugslayer) [inline README.md]: 4.0.0 is the latest stable official release :)
```
 Node.js 0.10.35 or a later stable 0.x release
```

File: README.md [hunk 3]
```diff
@@ -553,9 +595,9 @@ This software is licensed under the Apache 2.0 license, quoted <a href="LICENSE"
 [10]: http://developer.telerik.com/featured/nativescript-android/
 [11]: http://blogs.telerik.com/valentinstoychev/posts.aspx/14-06-12/announcing-nativescript---cross-platform-framework-for-building-native-mobile-applications
 [12]: https://developer.apple.com/xcode/downloads/
-[Node.js 0.10.26]: http://nodejs.org/download/
+[Node.js 0.10.35]: http://nodejs.org/download/
```
Comment (teobugslayer) [inline README.md]: https://nodejs.org/dist/v0.10.39/

File: README.md (hunk 1 above)
Comment (rosen-vladimirov) [inline README.md]: added few lines below

**Comment by teobugslayer** [comment]
:+1:

**Pull request merged**